### PR TITLE
Update store-data.md to correct warning

### PR DESCRIPTION
**Pull request by momack2**
similar to IPFS, these networks are find to use for encrypted data as long as you ack the security model: https://docs.ipfs.io/concepts/privacy/#encrypting-content-transported-via-ipfs

Maybe should be a warning instead of a "danger"?

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/get-started/store-and-retrieve/store-data.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://networks.filecoin.io
- The following link returned HTTP status code 404 (Not Found):
  https://plus.fil.org/miners/
- Insert a space between the numerical value and the unit symbol.
  ```
  ...tutorial, we're going to create a dummy 5GB file full of random data. Move i...
                                             ^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  .... Move into your home folder:  Create a 5GB block of random data to serve as...
                                             ^^^
  ```
- The operating system from Apple is written “macOS”.
  ```
  ...of random data to serve as our payload: MacOS users must run: Linux users sh...
                                             ^^^^^
  ```
- ‘For the sake of’ might be wordy. Consider a shorter alternative.
  ```
  ...storage providers that suit your needs. For the sake of this tutorial, look ...
                                             ^^^^^^^^^^^^^^^
  ```
- A word may be missing after ‘a’.
  ```
  ...orial, look for storage providers that: a. Offer verified-data deals for 0 F...
                                             ^^
  ```
- The verb ‘Are’ is missing a subject.
  ```
  ...Offer verified-data deals for 0 FIL. a. Are close to you. Once you have foun...
                                             ^^^
  ```
- ‘Miner’ seems less likely than ‘Minor’.
  ```
  ...ing to these IDs in the next section. # Miner reputation systems The Filecoi...
                                             ^^^^^
  ```
- Did you mean “to back up”?
  ```
  ...Specify the CID of the payload you want to backup on Filecoin. This is the C...
                                             ^^^^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...20 minutes to build the .car file of a ~7.5GB file with a 4-core CPU and 8GB...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ... of a ~7.5GB file with a 4-core CPU and 8GB RAM. These specifications are co...
                                             ^^^
  ```
- Consider shortening this phrase to just “whether”. It is correct though if you mean ‘regardless of whether’.
  ```
  ...n. The minimum is 180 days:  Tell Lotus whether or not this is a Filecoin Pl...
                                             ^^^^^^^^^^^^^^
  ```
- This phrase is redundant. Consider writing “space”.
  ```
  ...r IDs from the previous section with an empty space separating the two IDs: ...
                                             ^^^^^^^^^^^
  ```
- The modal verb ‘will’ requires the verb’s base form.
  ```
  ...  Confirm your transaction:  Lotus will returns two Deal CIDs:  Take a note ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... has reached StorageDealActive. See the Processing states table below to fin...
                                             ^^^^^^^^^^
  ```
- Consider using “many”.
  ```
  ...ilecoin network, deals can be in one of many different states. # Processing ...
                                             ^^^^^^^^^^^^^^
  ```
- Consider shortening this phrase to just “whether”. It is correct though if you mean ‘regardless of whether’.
  ```
  ...running custom decision logic to decide whether or not to accept the deal. T...
                                             ^^^^^^^^^^^^^^
  ```

File: docs/get-started/store-and-retrieve/store-data.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: Start storing your data on the Filecoin network. This section cover
 Start storing your data on the Filecoin network. This section covers packaging your data, importing it into your local Lotus lite-node, finding a storage provider through the Filecoin Plus miner registry, creating a storage deal, and then waiting for the deal to complete! 
 
 :::danger
-Do not store personal data on the Filecoin network, even if it's encrypted. Only store public and static data on the Filecoin network. Access control is on the project roadmap.
+Filecoin is optimized for public data and doesn't yet support access controls. If storing private data, ensure you encrypt it before storage to ensure it remains unreadable by anyone without the ability to decrypt it.
```
Comment (johnnymatthews) [inline docs/get-started/store-and-retrieve/store-data.md:11]:
```suggestion
Filecoin is optimized for public data and doesn't yet support access controls. If storing private data, ensure you encrypt it before storage to ensure it remains unreadable by anyone without the ability to decrypt it. If a vulnerability is found in your encryption process at any point in the future, then your data may be compromised.
```

Comment (johnnymatthews) [reply docs/get-started/store-and-retrieve/store-data.md:11]: It's important to stress that if a user encrypts their data with GPG, and then throws it on Filecoin and then an exploit in GPG is found, then that user's data could potentially be at risk.

Comment (momack2) [reply docs/get-started/store-and-retrieve/store-data.md]: fair point. added the suggestion

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/get-started/store-and-retrieve/store-data.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://networks.filecoin.io
- The following link returned HTTP status code 404 (Not Found):
  https://plus.fil.org/miners/
- Insert a space between the numerical value and the unit symbol.
  ```
  ...tutorial, we're going to create a dummy 5GB file full of random data. Move i...
                                             ^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  .... Move into your home folder:  Create a 5GB block of random data to serve as...
                                             ^^^
  ```
- The operating system from Apple is written “macOS”.
  ```
  ...of random data to serve as our payload: MacOS users must run: Linux users sh...
                                             ^^^^^
  ```
- ‘For the sake of’ might be wordy. Consider a shorter alternative.
  ```
  ...storage providers that suit your needs. For the sake of this tutorial, look ...
                                             ^^^^^^^^^^^^^^^
  ```
- A word may be missing after ‘a’.
  ```
  ...orial, look for storage providers that: a. Offer verified-data deals for 0 F...
                                             ^^
  ```
- The verb ‘Are’ is missing a subject.
  ```
  ...Offer verified-data deals for 0 FIL. a. Are close to you. Once you have foun...
                                             ^^^
  ```
- ‘Miner’ seems less likely than ‘Minor’.
  ```
  ...ing to these IDs in the next section. # Miner reputation systems The Filecoi...
                                             ^^^^^
  ```
- Did you mean “to back up”?
  ```
  ...Specify the CID of the payload you want to backup on Filecoin. This is the C...
                                             ^^^^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...20 minutes to build the .car file of a ~7.5GB file with a 4-core CPU and 8GB...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ... of a ~7.5GB file with a 4-core CPU and 8GB RAM. These specifications are co...
                                             ^^^
  ```
- Consider shortening this phrase to just “whether”. It is correct though if you mean ‘regardless of whether’.
  ```
  ...n. The minimum is 180 days:  Tell Lotus whether or not this is a Filecoin Pl...
                                             ^^^^^^^^^^^^^^
  ```
- This phrase is redundant. Consider writing “space”.
  ```
  ...r IDs from the previous section with an empty space separating the two IDs: ...
                                             ^^^^^^^^^^^
  ```
- The modal verb ‘will’ requires the verb’s base form.
  ```
  ...  Confirm your transaction:  Lotus will returns two Deal CIDs:  Take a note ...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... has reached StorageDealActive. See the Processing states table below to fin...
                                             ^^^^^^^^^^
  ```
- Consider using “many”.
  ```
  ...ilecoin network, deals can be in one of many different states. # Processing ...
                                             ^^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...xist, or your full-node is out of sync. StorageDealProposalRejected The stor...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- Consider shortening this phrase to just “whether”. It is correct though if you mean ‘regardless of whether’.
  ```
  ...running custom decision logic to decide whether or not to accept the deal. T...
                                             ^^^^^^^^^^^^^^
  ```

**Pull request merged**